auth-api/Dockerfile

```diff
@@ -7,6 +7,8 @@ ENV VCS_REF=${VCS_REF}
 ENV BUILD_DATE=${BUILD_DATE}
 ENV PORT=8080
 
+CMD gunicorn --bind 0.0.0.0:${PORT} --config /code/gunicorn_config.py wsgi:app
```

Comment:
Hmm this doesn't look right

Reply:
yeah we don't want this changed. this portion of the ticket is already done @sameer0422